config/app.ts

```diff
@@ -32,7 +33,7 @@ export const http = defineConfig({
     path: '/',
     maxAge: '2h',
     httpOnly: true,
-    secure: false,
-    sameSite: false,
+    secure: app.production,
```

Comment:
Is there a property called `app.production`? I think it is `inProduction`, but please double check